### PR TITLE
store: move DbVersion and DB_VERSION to near-store

### PR DESCRIPTION
**Pull request by mina86**
The database version definitions make more sense inside of near-store
than near-primitives.  Especially since majority of usage of them
is in near-store and the definitions aren’t used for anything specific
to the chain or other operations not involving the storage.

While at it, change RocksDB::get_version such that it checks whether the
database exists (and returns None if it doesn’t) and opens the database
with only DbVersion column configured (so that’ll properly read version
from old databases without modifying them).

### CODE REVIEW

**Comment by akhi3030** [comment]
@nagisa: do you maybe have some free cycles to review this PR please?

File: core/store/src/db/rocksdb.rs [hunk 1]
```diff
@@ -525,23 +524,27 @@ impl RocksDB {
...
+    /// database does not exist.
+    pub(crate) fn get_version(path: &Path, config: &StoreConfig) -> io::Result<Option<DbVersion>> {
+        if !path.join("CURRENT").is_file() {
+            return Ok(None);
+        }
+
+        // Specify only DBCol::DbVersion because it’s ok to open database in
+        // read-only mode without specifying all column families the database
+        // contains but it’s an error to provide a descriptor for column family
+        // which does note exist.
```
Comment (nagisa) [inline core/store/src/db/rocksdb.rs:537]:
```suggestion
        // which does not exist.
```

Comment (nagisa) [reply core/store/src/db/rocksdb.rs:537]: This comment could further justify why avoiding this error is desirable (e.g. making it possible to read out the database version regardless of which version the current database is at, thus avoiding errors before the migrations have concluded)

**Review by nagisa — COMMENTED** [review]
r=me once the comment is adjusted.

**Pull request merged**